sdk/nodejs/runtime/config.ts

```diff
@@ -53,6 +59,21 @@ export function getConfig(k: string): string | undefined {
     return config[k];
 }
 
+/**
+ * isConfigSecret returns true if the key contains a secret value.
+ * @internal
+ */
+export function isConfigSecret(k: string): boolean {
+    const envConfigSecretKeys = process.env[configSecretKeysEnvKey];
+    if (envConfigSecretKeys) {
+        const envConfigSecretArray = JSON.parse(envConfigSecretKeys);
+        if (Array.isArray(envConfigSecretArray)) {
+            return envConfigSecretArray.includes(k);
```

Comment:
Does this need to handle path keys like `foo.bar` or `foo[0]`?

Reply:
No, we don't have any APIs in the SDKs for accessing leaf nodes by path. Such values must be accessed via the top-level key with `getObject`, `requireObject`, `getSecretObject`, or `requireSecretObject`.